lib/dnsimpler/http.rb

```diff
@@ -43,7 +43,7 @@ def self.#{method}(path, opts = {}, &blk)
             if DNSimpler.debug
               response.request = req
 
-              puts "Request Options: " + opts.to_s
+              $stdout.puts "Request Options: #{opts.inspect}" 
```

Comment:
This causes the test to fail when the`opts` are nil. `opts` probably shouldn't ever be nil though.